api/src/test/java/uk/gov/dhsc/htbhf/claimant/repository/PaymentCycleRepositoryTest.java

```diff
@@ -216,8 +232,8 @@ private Claim createAndSaveClaim() {
         return claim;
     }
 
-    private Claim createAndSaveClaimWithNino(String nino) {
-        Claim claim = aClaimWithNinoAndClaimStatus(nino, ClaimStatus.ACTIVE);
+    private Claim createAndSaveClaimWithNino(String nino, String reference) {
+        Claim claim = aClaimWithNinoAndClaimStatus(nino, ClaimStatus.ACTIVE, reference);
```

Comment:
Rename 'aClaimWithNinoAndClaimStatus' to 'aClaimWithNinoAndClaimStatusAndReference'